_docs/tasks/security/per-service-mtls.md

```diff
@@ -53,6 +53,11 @@ kubectl exec $(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name}
 command terminated with exit code 56
 ```
 
+## Disable mutual TLS authentication for "httpbin" service using authentication policy.
```

Comment:
Do you plan to replace this doc with the authentication policy task doc?

Reply:
Yes, indeed. I remove this section and add note in the beginning of this task to refer to authn policy task. I think we will need to rewrite all these once we got authN policy working smoothly.